src/main/java/com/conveyal/datatools/manager/DataManager.java

```diff
@@ -200,7 +200,7 @@ private static void registerRoutes() throws IOException {
         });
 
         InputStream auth0Stream = DataManager.class.getResourceAsStream("/public/auth0-silent-callback.html");
-        String auth0html = IOUtils.toString(auth0Stream);
+        final String auth0html = IOUtils.toString(auth0Stream);
```

Comment:
I just meant that as a String that is predefined and hardwired and will not change between instances of the class (it's a constant), the convention is to make it static and final. Adding the final keyword is all you can do in this context/scope inside a method, although by that line of reasoning we could/should add many other final keywords all over the place. But usually what you'd do is pull a constant field like this out to the outermost level of the class and declare it `static final`, with an all caps identifier. This is just Java convention for a symbolic constant.